### PR TITLE
Add Gunicorn to LWRP

### PR DESCRIPTION
**Pull request by tschuy**
This will allow a django app cookbook to include a port and automatically set up supervisor and gunicorn.

There are integration and unit tests included.

### CODE REVIEW

File: spec/unit/default_spec.rb [hunk 1]
```diff
@@ -70,6 +70,39 @@
...
+    expect(chef_run)
+      .to enable_supervisor_service('whats_fresh').with(
+        command: '/opt/venv_whats_fresh/bin/gunicorn' \
+          ' whats_fresh.wsgi:application' \
+          ' -c /opt/whats_fresh/gunicorn_config.py',
+        autorestart: true,
+        directory: '/opt/whats_fresh')
+  end
+
+  it 'installs gunicorn to Whats Fresh virtualenv' do
```
Comment (mathuin) [inline spec/unit/default_spec.rb]: This is a generic project cookbook, but it makes specific references here and elsewhere to What's Fresh.  Is this intentional?

Comment (iankronquist) [inline spec/unit/default_spec.rb]: This is an artifact from an earlier stage in development, it used to check out What's Fresh and Working waterfronts and set them up, but then we ran into issues configuring PostGres, so we dumped them for a generic cookbook.
I'm not sure if we should refactor this. I'm not sure what we would rename them to, perhaps 'project-a', 'project-b', etc.?

Comment (mathuin) [inline spec/unit/default_spec.rb]: It sounds like the cookbook requires a webapp for testing.

I would consider building the "hello world" Django app from the tutorial: https://docs.djangoproject.com/en/1.8/intro/tutorial01/  This has the advantage of being a) real, b) familiar, and c) very likely to be updated and maintained by the Django folks.  The only thing better would be an official version we could reference here on github, but those folks don't host a copy.  You could then name it 'tutorial-a', and use it for actual integration testing.  What do you think?

Comment (iankronquist) [inline spec/unit/default_spec.rb]: That's what we do, we use this repo: https://github.com/osuosl-cookbooks/osl-application-python

Comment (iankronquist) [inline spec/unit/default_spec.rb]: This has been fixed on the common_recipe branch. @tschuy should rebase this branch.

Comment (tschuy) [inline spec/unit/default_spec.rb]: I've rebased.

**Comment by iankronquist** [comment]
Tests pass, it's been rebased, this looks good to me.

**Pull request merged**